### ISSUE TITLE
Font does not switch immediately

### ISSUE THREAD

**Issue by GunnarBernsteinHH** [gsantner/markor#2462]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a [question or feature request](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The bug is still present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Marder**. Don't worry; Markor and Marder appear as completely separate applications. You can install both side-by-side, and Markor settings are not touched. In case the issue is resolved there, you don't need to create a bug report. The change will be part of the next Markor update.)

### Description

I am in the Quicknote Editor. Then I go into the More/Mehr - Editor - Settings and change the font. When I go back to Quicknote, no change can be seen. I can even switch between editors. 
Only when I reload the Note from File System, the change occurs. I might not have saved yet.
There is a refresh missing.

I also would like the font to be used in the View, which is not the case with AsciiDoc, but this may be a feature.

### Steps to reproduce

1. Open Quicknote, type some lines
2. Go Back, Go to More - Settings - Font
3. Change Font, e.g. Monospace
4. Go Back to Quicknote

### Information

Android version: 
Device: 
App Version: 

### Source

Google Play

### Format / File type

Plaintext

### Additional info / Log

```shell
Sorry, did not install Marder.
```

**Comment by gsantner** [comment]
Yes, some settings are only applied fully upon restart. Though it's rather things that you don't change frequently.

**Comment by thc282** [comment]
beside the font
the font size is same

the menu can switch the font size
but that only changing the font size in edit mode
the preview mode will not follow

preview can only change inside the setting

**Comment by gsantner** [comment]
preview mode is fully customizable with CSS.
